test/integration/stepper/verticalStepper.spec.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-env mocha */
+import React from 'react';
+import {mount} from 'enzyme';
+import {assert} from 'chai';
+import getMuiTheme from 'src/styles/getMuiTheme';
+import VerticalLinearStepper from '../fixtures/VerticalLinearStepper';
```

Comment:
@oliviertassinari I don't think that's a good idea, for 2 reasons:

1. It creates a dependency where one might not expect it. Whether it's docs structure, example specifics, etc... It would be different if a test was **based** on the entire documentation site for something like full e2e integration testing. But these types of integration tests might not always be so close to a docs example, and in that case you would end up with a fractured structure (some docs, some fixtures).
2. I have code in that fixture specifically for testing, eg:
```jsx
// attributes for easy selection with `enzyme`
<p testContent={1}>
```

and

```jsx
// duration is very low so testing isn't time consuming
<StepContent transitionDuration={10}>
```